### PR TITLE
Mobile Web 2022 queries

### PR DESCRIPTION
**Pull request by siakaramalegos**
Copies 2021 queries and updates the dates to 2022

Related to #2893

### CODE REVIEW

File: sql/2022/mobile-web/README.md [hunk 1]
```diff
@@ -1,20 +1,12 @@
...
-
-## Resources
-
-- [📄 Planning doc][~google-doc]
-- [📊 Results sheet][~google-sheets]
-- [📝 Markdown file][~chapter-markdown]
-
-[~google-doc]: https://docs.google.com/document/d/1CbfJi1AvEJp3N8h_oL6aUJfSroQm0WbzGFIWcRz1dQA/edit?usp=sharing
-[~google-sheets]: https://docs.google.com/spreadsheets/d/1GB39gbyOilTSvgJHVdY8zwxwaFdsUB5iaD29cMwkdDY/edit?usp=sharing
-[~chapter-markdown]: https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/src/content/en/2022/mobile-web.md
```
Comment (rviscomi) [inline sql/2022/mobile-web/README.md:20]: Worth keeping these resources

File: sql/2022/mobile-web/README.md [hunk 2]
```diff
@@ -1,20 +1,12 @@
 # 2022 Mobile Web queries
 
+This is a copy of 2021 queries (no new list).
+
 <!--
-  This directory contains all of the 2022 Mobile Web chapter queries.
+  This directory contains all of the 2021 Mobile Web chapter queries.
```
Comment (rviscomi) [inline sql/2022/mobile-web/README.md:6]: These reference 2022 data so they're still technically 2022 queries.

```suggestion
  This directory contains all of the 2022 Mobile Web chapter queries.
```

Comment (siakaramalegos) [reply sql/2022/mobile-web/README.md:6]: Oh weird didn't realize I overwrote stock content

**Comment by siakaramalegos** [comment]
@rviscomi made the fixes. I'm not authorized to merge

**Review by rviscomi — APPROVED** [review]

**Pull request merged**